### ISSUE TITLE
the results on Cifar10

### ISSUE THREAD

**Issue by danczs** [facebookresearch/ResNeXt#9]
I trained ResNeXt-64x8 and ResNeXt-64x16 on Cifar-10. But the result is slight  worse than the results released in the paper. I trained these models by three gpus:
th main.lua -dataset cifar10 -bottleneckType resnext_C -depth 29 -baseWidth 64 -cardinality 16/8 -weightDecay 5e-4 -batchSize 128 -nGPU 3 -nThreads 8 -shareGradInput true
the best model results I got for each run is: 
ResNeXt-64x8: 3.68%, 3.8%, 3.86%  3.90%  3.76%  3.84%
ResNeXt-64x16: 3.84%  3.60%
The average results in the paper is 3.65% and 3.58% for these two net.
Does someone have the same question?  Any suggestions? 
Thanks a lot.

**Comment by s9xie** [comment]
If you train with different number of GPUs, you need to tune the LR as well. We provided the configuration for using 2, 4, or 8 gpus. You should be able to reproduce the results using our provided configuration with 2 GPUs.

**Comment by danczs** [comment]
Thanks for your response. But  my 3 gpus can run the models with the batchsize of 128. Is small batchSize(32 or 64) the only reason why you tune the LR on 2 and 4 GPUs?

**Comment by s9xie** [comment]
It is because the batchsize and learning rate are correlated: to learn more about effective learning rate and linear scaling rule, please refer to https://arxiv.org/pdf/1706.02677.pdf

**Comment by danczs** [comment]
Thanks a lot.
